From 013a5717611dd76fc0110747eca1057042e7eeb7 Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Mon, 4 Jul 2022 17:43:37 -0400
Subject: [PATCH] nezha: add pwm LED to dts

---
 arch/riscv/dts/sun20i-d1-nezha.dts | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/arch/riscv/dts/sun20i-d1-nezha.dts b/arch/riscv/dts/sun20i-d1-nezha.dts
index ce579430..66d1f869 100644
--- a/arch/riscv/dts/sun20i-d1-nezha.dts
+++ b/arch/riscv/dts/sun20i-d1-nezha.dts
@@ -62,6 +62,21 @@
 		compatible = "mmc-pwrseq-simple";
 		reset-gpios = <&gpio 6 12 GPIO_ACTIVE_LOW>; /* PG12 */
 	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		led@2 {
+			reg = <0x2>;
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_INDICATOR;
+			pwms = <&pwm 2 50000 0>;
+			max-brightness = <255>;
+		};
+	};
 };
 
 &codec {
-- 
2.25.1

